services/metadata_service/tests/unit_tests/api_util_test.py

```diff
@@ -0,0 +1,23 @@
+import json
+from services.metadata_service.api.utils import handle_exceptions, format_response
+
+async def test_handle_exceptions():
+
+    @handle_exceptions
+    async def do_not_raise():
+        return True
+
+    @format_response
+    @handle_exceptions
+    async def raise_without_id():
+        raise Exception("test")
+
+    # wrapper should not touch successful calls.
+    assert (await do_not_raise())
```

Comment:
I'm guessing the addition of `@format_response` was to get this test to pass? It has the unwanted effect that we are also testing the format_response implementation in a single unit test. Not a blocker for merging of course, and can be addressed in #146 later on.

Reply:
considering `@handle_exceptions` is only ever used when accompanied by  `@format_response` I don't think this is a problem.